frontend-utils/src/backends/navigator.rs

```diff
@@ -182,26 +182,13 @@ impl<F: FutureSpawner + 'static, I: NavigatorInterface> NavigatorBackend
             return;
         }
 
-        if self.open_url_mode == OpenURLMode::Confirm {
-            if !self.interface.confirm_website_navigation(&modified_url) {
-                tracing::info!("SWF tried to open a website, but the user declined the request");
-                return;
-            }
-        } else if self.open_url_mode == OpenURLMode::Deny {
+        if self.open_url_mode == OpenURLMode::Deny {
             tracing::warn!("SWF tried to open a website, but opening a website is not allowed");
             return;
         }
 
-        // If the user confirmed or if in Allow mode, open the website
-
-        // TODO: This opens local files in the browser while flash opens them
-        // in the default program for the respective filetype.
-        // This especially includes mailto links. Ruffle opens the browser which opens
-        // the preferred program while flash opens the preferred program directly.
-        match webbrowser::open(modified_url.as_ref()) {
-            Ok(_output) => {}
-            Err(e) => tracing::error!("Could not open URL {}: {}", modified_url.as_str(), e),
-        };
```

Comment:
This was a nice built-in default behavior that I now have to duplicate in the Android app.
It's literally 3 lines of code, so it's not a big deal - just thought I'd mention it (to potentially continue encouraging the consideration of the "official" alternative - out-of-repo - front-end implementation).

Reply:
I did it being fully aware that it'll duplicate this call on Android.

The alternatives of either introducing `async` here, spawning a future, or passing a callback seemed like an overkill. I also took into account that we would like to open an app if a link points to a local file, which I assumed would have a vastly different implementation on Android vs desktop. In that case orchestrating everything by making a call here and synchronizing on the response just to invoke another method shortly afterwards seemed silly.

I could have introduced a utility method that calls `webbrowser::open`, but its only responsibility would be to just call it and possibly log the error, which also seemed like an overkill to use frontend-utils for that.

Reply:
Also I doubt you can make blocking UI on Android (which would have to be the case without this refactor), am I right?

Reply:
> I did it being fully aware that it'll duplicate this call on Android.

That's cool then! ^^

> The alternatives of either introducing `async` here, spawning a future, or passing a callback seemed like an overkill. 

Yeah, I agree.

> Also I doubt you can make blocking UI on Android (which would have to be the case without this refactor), am I right?

I have no idea honestly, but it's not relevant either then.